app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -64,4 +64,93 @@ describe("the new project page", () => {
     // Renders the default error message for a required field
     cy.get(".error-detail").last().should("contain", "Please enter a value");
   });
+
```

Comment:
Line 26 of this file just needs a selector fix:
cy.get('#primaryContactForm_contactId').click()